### DISCUSSION TITLE
Day Today GB

### DISCUSSION THREAD

**Discussion by TheUKHighStreet** [alltheplaces/alltheplaces#10941] [General]
The Day Today storefinder is not accurate. It lists several stores as open which are actually now a different brand (I haven't been through the whole thing - I've just come across several cases). I realise that this is the problem of the store finder on their website but I'm not sure whether it should be noted anywhere?
Example: 32b Main Street, Cowie, Stirling, FK7 7BL is actually a Londis.
The Scottish Tobacco Register lists the active and inactive stores at that address: https://www.tobaccoregisterscotland.org/search-the-register/?Postcode=FK7+7BL

**Comment by CloCkWeRX** [comment]
#5671 https://www.day-today.co.uk/shops/

**Comment by CloCkWeRX** [comment]
Bit of a tricky one, as they list day today, day today express, day today elite from a cursory scroll through - no other brands.

**Comment by TheUKHighStreet** [comment]
I think they just haven't removed stores which have changed to a different brand so the store finder is out of date.

**Reply by matkoniecz** [reply]
I guess that looking are they are somehow marked and can be filtered is useful.

Or maybe just wait a bit for an update? How outdated it is? By days? Weeks? Months? Years?

**Reply by TheUKHighStreet** [reply]
I think that the fact that I've come across several suggests it is months rather than weeks. But looking at the one example I've given, the FSA inspection of the Londis store was on 04 April 2022 - so I think it is actually years.

**Reply by matkoniecz** [reply]
at this point I would consider removing spider as emitting bogus data

**Reply by TheUKHighStreet** [reply]
Maybe it needs an estimate of what percentage is bogus as @CloCkWeRX suggested - but I'm not sure how long that would take to work out.

**Reply by matkoniecz** [reply]
checking say sample of 50 random location?

**Reply by TheUKHighStreet** [reply]
That's what I was thinking but some of them may not be obviously wrong. Will have a look tomorrow.

**Reply by TheUKHighStreet** [reply]
I've had a look at 50 random stores - quite a few were difficult to decide whether they really were Day Today but 5 out of 50 seemed to definitely be other companies. So that's at least a 10% failure rate (and probably higher).
https://docs.google.com/spreadsheets/d/1f-5mEBV8TLB5XfNkh8lv6R1entJHqnhHZvpvodcXV8A/edit?usp=sharing

**Reply by matkoniecz** [reply]
I opened https://github.com/alltheplaces/alltheplaces/pull/10974 and ignored it in my OSM-ATP matcher.

**Comment by CloCkWeRX** [comment]
I think there is not much we can do while still trusting them at all as a data source.

We're basically publishing structured data at about where the "dictionaries" are is this diagram, and relying on human verification (survey/street view imagery/or your cross checking of registers) to catch the outdated or false claims (proof, trust)

![17283075670811374527825477104666](https://github.com/user-attachments/assets/080df0e4-3c19-49cb-80a7-28b206ca633d)

Maybe worth adding comments to the spider itself to indicate it's not always reliable, link to this thread?

Longer term the alternative would be to add some kind of annotations about how we perceive the data quality in the output ("bad coordinates", "incomplete hours", "prone to fake locations for SEO"), so it's easier for downstream tools to decide how to handle.

@matkoniecz is building a lot of quality checks on low level attributes; in addition to that I could see value in having a second set of spiders for a lot of gov data (alcohol, tobacco, food safety/restaurant licencing) doing cross comparisons as you have done... But I'm not sure how consistent their schemas are.

**Reply by matkoniecz** [reply]
> Maybe worth adding comments to the spider itself to indicate it's not always reliable, link to this thread?

> Longer term the alternative would be to add some kind of annotations about how we perceive the data quality in the output ("bad coordinates", "incomplete hours", "prone to fake locations for SEO"), so it's easier for downstream tools to decide how to handle.

appeared in say https://github.com/alltheplaces/alltheplaces/issues/6818

**Reply by TheUKHighStreet** [reply]
Yes - even the government databases aren't alway that accurate - I'm often flagging up inaccuracies in the government gambling licences database and the NHS databases. The NHS at least does update reasonably quickly once you flag a problem.

**Reply by CloCkWeRX** [reply]
> data_quality:position=inaccurate or data_quality:known_offset=~300m

I like that, would want to make sure there is a check_date or similar though.

Also worth a look at:
https://www.w3.org/TR/vocab-dqv/#ExpressAQualityAssessmentWithQualityMetrics
Probably has some ideas and terms we haven't looked at for general statements of how good a dataset is.

Edit: okay, that's a bit general. Expected completeness of set as a % is my main takeaway.

**Comment by CloCkWeRX** [comment]
TODO: 
https://wiki.openstreetmap.org/wiki/Tag:shop%3Dtobacco suggests that 'ref' or 'ref:tobacco' is used to cross reference licences.
Should look at open data exports or similar containing licence numbers and address info in more detail; possibly link to open corporates or some other business register where the brand/trading name is not the legal entity.
